docs/getting-started/tutorial/index.md

```diff
@@ -0,0 +1,16 @@
+In the tutorial, we will create a study that uses dummy patient data.
+We will:
+
+1. Crete a GitHub account
+2. Create a new repository
+3. Create a study
+4. Run the study in a local environment
+5. Update the study
+6. Run the study in a testing environment
```

Comment:
nit: I'd use `1.` for each item, because it means that you don't have to ever worry about renumbering the list, should items get moved or removed.

```suggestion
1. Create a new repository
1. Create a study
1. Run the study in a local environment
1. Update the study
1. Run the study in a testing environment
```

Reply:
Will do 👍🏻